app/src/debug/java/com/artemzin/qualitymatters/ui/fragments/DeveloperSettingsFragment.java

```diff
@@ -197,6 +200,11 @@ public void showAppNeedsToBeRestarted() {
         runOnUiThreadIfFragmentAlive(() -> Toast.makeText(getContext(), "To apply new settings app needs to be restarted", Toast.LENGTH_LONG).show());
     }
 
+    @OnClick(R.id.tv_show_log)
+    void showLog(View view) {
+        view.getContext().startActivity(LynxActivity.getIntent(view.getContext(), new LynxConfig()));
```

Comment:
Nice library! I'll take a look tonight. Please create an issue to discuss
things before PR next time :)

On Tue, Dec 29, 2015, 13:56 Niklas Baudy <notifications@github.com> wrote:

> In
> app/src/debug/java/com/artemzin/qualitymatters/ui/fragments/DeveloperSettingsFragment.java
> <https://github.com/artem-zinnatullin/qualitymatters/pull/77#discussion_r48533530>
> :
>
> > @@ -197,6 +200,11 @@ public void showAppNeedsToBeRestarted() {
> >          runOnUiThreadIfFragmentAlive(() -> Toast.makeText(getContext(), "To apply new settings app needs to be restarted", Toast.LENGTH_LONG).show());
> >      }
> >
> > +    @OnClick(R.id.tv_show_log)
> > +    void showLog(View view) {
> > +        view.getContext().startActivity(LynxActivity.getIntent(view.getContext(), new LynxConfig()));
>
> Also should not we provide the LynxConfig via Dagger?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/artem-zinnatullin/qualitymatters/pull/77/files#r48533530>
> .
>
-- 

@artem_zin